README.md

```diff
@@ -53,143 +46,157 @@ CPG Flow supports various stages of genomic data processing, from raw data inges
 
 ### 🌐 Production
 
-The production version of this API is available at **[api.werewolves-assistant.com](https://api.werewolves-assistant.com/docs)**.
-
-This API is used by the **[Werewolves Assistant Web App](https://werewolves-assistant.com)**.
-
-The production server is updated automatically with the latest version of the API when a new release is created. (When a new tag is pushed on the `main` branch)
+The production version of this API is documented at **[populationgenomics.github.io/cpg-flow](https://populationgenomics.github.io/cpg-flow/main/cpg_flow.html)**.
 
 ### 🛠️ Development
 
-The development version of this API is available at **[preprod.api.werewolves-assistant.com](https://preprod.api.werewolves-assistant.com/docs)**.
-
-This API is used by the **[Werewolves Assistant Web App](https://preprod.werewolves-assistant.com)**.
+The development version of this API is available at **[populationgenomics.github.io/cpg-flow](https://populationgenomics.github.io/cpg-flow/alpha/cpg_flow.html)**.
```

Comment:
Maybe we should change the hyperlink text so it reflects the hyperlinks themselves

Reply:
Ah yes good idea. That's legacy because there used to just be one doc page. I'll fix it up.